recipes/pixz/recipe.yaml

```
@@ -0,0 +1,44 @@
+context:
+  version: 1.0.7
+
+package:
+  name: pixz
+  version: ${{ version }}
+
+source:
+  url: https://github.com/vasi/pixz/releases/download/v${{ version }}/pixz-${{ version }}.tar.gz
+  sha256: d1b6de1c0399e54cbd18321b8091bbffef6d209ec136d4466f398689f62c3b5f
+
+build:
+  number: 0
+  skip: win
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - autoconf
+    - automake
+    - gettext
+    - make
+    - pkg-config
+    - sed
+  host:
+    - libarchive
+    - libtool
```

For gettext support on osx, this is normally required.
```suggestion
    - libtool
    - libintl-devel  # [osx]
```